application/src/main/java/org/togetherjava/tjbot/commands/utils/MessageUtils.java

```diff
@@ -54,11 +54,12 @@ public static void disableButtons(@NotNull Message message) {
      * @return the generated embed
      */
     public static @NotNull MessageEmbed generateEmbed(@Nullable String title,
-            @NotNull CharSequence content, @Nullable User user, @Nullable Color ambientColor) {
+            @NotNull CharSequence content, @Nullable User user, @Nullable Color ambientColor,
+            @Nullable String footerTag) {
```

Comment:
I think this is overly specific. Change the role of `footerTag` to just a `footer`. And then move the `user.getName() + footerTag` logic into the call-site.

So that all callers can now use this parameter as footer and not as "footerTag".